pyproject.toml

```diff
@@ -17,7 +17,9 @@ dependencies = [
     "nslsii",
     "pyepics",
     "aioca",
-    "pydantic<2.0",        # Leave pinned until can check incompatibility
+    "pydantic>=2.0",
+    "scanspec>=0.7.0",
```

Comment:
Since we know scanspec 0.7.0 isn't compatible with serialising BaseModels like we need to, let's keep it at >=0.7.1